### DISCUSSION TITLE
Custom design

### DISCUSSION THREAD

**Discussion by DeyvidA** [wojtekmaj/react-calendar#596] [Q&A] [answered]
How i can add my own disegin, colors etc ?

**Comment by laocker96** [comment]
Just override react-calendar css

**Comment by wojtekmaj** [comment, marked as answer]
Please see https://github.com/wojtekmaj/react-calendar#custom-styling

You could use this file as your "base", or copy it to your repo and start editing it to suit your needs, or build something entirely from scratch.

Have a look at the discussion regarding styles we're using on the demo page, too: https://github.com/wojtekmaj/react-datetime-picker/issues/90#issuecomment-805719021
